### PR TITLE
Review only PR

### PR DESCRIPTION
**Pull request by mmikev**

### CODE REVIEW

File: packages/constants/src/chain.ts [hunk 1]
```diff
@@ -1,14 +1,20 @@
 import { NETWORK } from './network';
 
+export enum MAINNET_CHAINS {
+  COLUMBUS4 = 'columbus-4',
```
Comment (zavgorodnii) [inline packages/constants/src/chain.ts]: do we really need anything columbus-4 related?

File: packages/constants/src/chain.ts [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+  COLUMBUS5 = 'columbus-5',
+}
+
+export enum TESTNET_CHAINS {
+  BOMBAY11 = 'bombay-11',
```
Comment (zavgorodnii) [inline packages/constants/src/chain.ts]: same here

File: packages/facade/src/contracts/hub.ts [hunk 3]
```diff
@@ -222,7 +239,7 @@ class LidoTerraHub extends LidoTerraBaseContract {
       withdrawable_unbonded: { address },
     });
     return {
-      withdrawableUnbonded: new Int(response.withdrawable_unbonded),
+      withdrawableUnbonded: new Int(response.withdrawable),
```
Comment (swelf19) [inline packages/facade/src/contracts/hub.ts]: line does not follow "code convention" where model's field name in camel case = response field name in snake case

File: packages/constants/src/chain.ts (hunk 1 above)
Comment (ratik) [reply packages/constants/src/chain.ts]: removed 👍

File: packages/constants/src/chain.ts (hunk 2 above)
Comment (ratik) [reply packages/constants/src/chain.ts]: removed 👍

File: packages/facade/src/contracts/hub.ts (hunk 3 above)
Comment (ratik) [reply packages/facade/src/contracts/hub.ts]: renamed 👍

**Review by zavgorodnii — APPROVED** [review]

**Review by swelf19 — APPROVED** [review]

**Pull request closed without merging**